### PR TITLE
Add Sony Entertainment shared credentials backend

### PR DESCRIPTION
**Pull request by nicklasholm**
Fixes apple/password-manager-resources#75

### CODE REVIEW

**Comment by nicklasholm** [comment]
Fixes apple/password-manager-resources#75

File: quirks/websites-with-shared-credential-backends.json [hunk 1]
```diff
@@ -217,5 +217,9 @@
     [
         "lookmark.io",
         "lookmark.link"
+    ],
+    [
+        "playstation.com",
+        "id.sonyentertainmentnetwork.com"
```
Comment (rmondello) [inline quirks/websites-with-shared-credential-backends.json:223]: Are there login pages served from both of these domains that accept the same credentials?

Comment (DerJuulsn) [reply quirks/websites-with-shared-credential-backends.json:223]: playstation.com will forward you to sony.com if you want to sign in

Comment (rmondello) [reply quirks/websites-with-shared-credential-backends.json:223]: Then maybe this PR isn’t valid or needed.

Comment (nicklasholm) [reply quirks/websites-with-shared-credential-backends.json:223]: This was just a fix for issue #75, feel free to close if not valid.

Comment (rmondello) [reply quirks/websites-with-shared-credential-backends.json:223]: I appreciate you giving this a go, @nicklasholm. :)

**Pull request closed without merging**